packages/velo-external-db-core/lib/router.js

```diff
@@ -26,27 +28,36 @@ const initServices = (_schemaAwareDataService, _schemaService, _operationService
     filterTransformer = _filterTransformer
     aggregationTransformer = _aggregationTransformer
     roleAuthorizationService = _roleAuthorizationService
-    hooks = _hooks || {}
+    dataHooks = _hooks?.dataHooks || {}
+    schemaHooks = _hooks?.schemaHooks || {}
 }
 
-const serviceContext = {
+const serviceContext = () => ({
```

Comment:
I think since it's a function you better rename this to something with a verb, like `getServiceContext`